packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -105,67 +106,67 @@ export class SDKClient {
                     const accountBalance = await (new AccountBalanceQuery()
                         .setAccountId(clientMain.operatorAccountId!))
                         .execute(clientMain);
```

Comment:
nit: class method `getAccountBalance` can be used here.

Reply:
Thanks will address this in follow up